proposed/dotnet-bootstrapper-cli-experience.md

```diff
@@ -0,0 +1,256 @@
+# DNVM / Dotnet Bootstrapper End-to-End CLI Experience
+
+Initial Context:
+
+The officially supported `DNVM` and `DNX` project was replaced by the `.NET CLI` as `dotnet` in `2017`.
+
+As of 2025, `DNVM` is an unofficial product, but internally developed and leveraged.
+
+We are proposing an official `DNVM` product (name subject to change).
+This is an offering similar to `nvm` or `rustup` but for `.NET`.
+There have already been many internal documents/discussions. This is a 'strawman proposal' for us to dig at further. For all mentions of the term `dnvm`, this is subject to change to an applicable product name.
+
+The `dnvm` product will be AOT bundled alongside the apphost to enable shipment via an external standalone executable, that is also callable from within the .NET SDK. The implementation details of this will be settled and discussed within another document, but it's necessary to remark on this here to explain the usage of 'dotnet' as a root command.
+
+## 🗃️ Installation Experience
+
+```
+dotnet install
+dotnet install <channel>
+```
+
+### MVP:
+`dotnet install` → installs latest STS/LTS SDK, or the 'latest' acceptable SDK based on `global.json` if specified. The installation modifies the user-level `PATH`. (This is complicated and will get its own document.)
```

Comment:
Is the thinking here along the lines of [actions/setup-dotnet](https://github.com/actions/setup-dotnet), so I could `cd project && dotnet install` and it would install whatever matches `project/global.json`?

Reply:
Yes. @richlander had a great point that we should try to align some of the behaviors of the github action here.

Reply:
That's a great addition, I will add this context.